content/telegraf/v1/install.md

```diff
@@ -100,68 +99,44 @@ To manually install the Debian package from a `.deb` file:
    sudo dpkg -i telegraf_{{< latest-patch >}}-1_amd64.deb
    ```
 
-{{% telegraf/verify %}}
-
-## Configuration
-
-### Create a configuration file with default input and output plugins.
-
-Every plugin will be in the file, but most will be commented out.
-
-```
-telegraf config > telegraf.conf
-```
-
-### Create a configuration file with specific inputs and outputs
-```
-telegraf --input-filter <pluginname>[:<pluginname>] --output-filter <outputname>[:<outputname>] config > telegraf.conf
-```
-
-For more advanced configuration details, see the
-[configuration documentation](/telegraf/v1/administration/configuration/).
 {{% /tab-content %}}
 <!---------- BEGIN RedHat & CentOS ---------->
 {{% tab-content %}}
-For instructions on how to manually install the RPM package from a file, please see the [downloads page](https://influxdata.com/downloads/).
-
-**RedHat and CentOS:** Install the latest stable version of Telegraf using the `yum` package manager:
-
-```bash
-cat <<EOF | sudo tee /etc/yum.repos.d/influxdb.repo
-[influxdb]
-name = InfluxData Repository - Stable
-baseurl = https://repos.influxdata.com/stable/\$basearch/main
-enabled = 1
-gpgcheck = 1
-gpgkey = https://repos.influxdata.com/influxdata-archive_compat.key
-EOF
-```
-
-Install telegraf once the repository is added to the `yum` configuration:
-
-```bash
-sudo yum install telegraf
-```
+To learn how to manually install the RPM package from a file, see the [downloads page](https://influxdata.com/downloads/).
+
+Use the `yum` package manager to install the latest stable version of Telegraf:
+
+1. In the [InfluxData package repository](https://repos.influxdata.com/stable/), find your system architecture name in the file list (for example, `x86_64`), and then copy the repository URL for your architecture (for example, https://repos.influxdata.com/stable/x86_64).
```

Comment:
Why are you using this approach or the existing?

If the user uses the `\$basearch` these steps are handled for them by yum and there is no need for them to navigate the repo website or export the baseurl variable.

Reply:
> Why are you using this approach or the existing?
> 
> If the user uses the `\$basearch`

My lack of understanding. I didn't realize `$basearch` was a yum variable. Thanks!